db/schema/item.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+const user = require('./user');
+
+const itemSchema = new mongoose.Schema({
+    id: String,
+    date: String,
+    tax: Number,
+    tip: Number,
+    currency: {
+        type: String,
+        default: 'USD'
+    },
+    total: Number,
+    creator: user,

```

Comment:
Item schema also does not need total or creator for the same reasons as the previous comment. Instead, the fields for item name, array of people associated with the item, and cost of item need to be included